### PR TITLE
Right side view of binary Tree in java

### PR DESCRIPTION
**Pull request by krishnmohan19**

### CODE REVIEW

File: Code/Java/Right_Side_View_Of_BinaryTree.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+{
+    int key;
+    Node left = null, right = null;
+ 
+    Node(int key) {
+        this.key = key;
+    }
+}
+ 
+class Main
```
Comment (raksha009) [inline Code/Java/Right_Side_View_Of_BinaryTree.java:33]:
```suggestion
class Right_Side_View_Of_BinaryTree
```

File: Code/Java/Right_Side_View_Of_BinaryTree.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                if (curr.right != null) {
+                    queue.add(curr.right);
+                }
+            }
+        }
+    }
+ 
+    public static void main(String[] args)
+    {
+        Node root = new Node(1);
```
Comment (raksha009) [inline Code/Java/Right_Side_View_Of_BinaryTree.java:81]: Plz take input dynamically!

**Review by raksha009 — CHANGES_REQUESTED** [review]

**Comment by krishnmohan19** [comment]
hey @raksha009 ,
i have made changes .plz review it.

File: Tree/readme.md [hunk 3]
```diff
@@ -51,6 +51,7 @@ that, this property should be satisfied at every node in the tree.
 * Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
 * Construction of binary tree from preorder and inorder traversal ----> [C++](/Code/C++/binary_tree_from_preorder_and_inorder.cpp) 
 * Level Order Traversal in BST ----> [C++]() | [Java]() | [Python](/Code/Python/level_order_traversal_binary_tree.py)
+* Right Side View of Binary Tree ----> [JAVA](/Code/java/Right_Side_View_Of_BinaryTree.java)
```
Comment (raksha009) [inline Tree/readme.md:54]:
```suggestion
* Right Side View of Binary Tree ----> [Java](/Code/java/Right_Side_View_Of_BinaryTree.java)
```

**Review by raksha009 — CHANGES_REQUESTED** [review]
Just make this small change, rest LGTM

**Comment by raksha009** [comment]
Also, resolve the conflicts :)

**Comment by tarun26091999** [comment]
#1061

**Pull request merged**